riocli/usergroup/model.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2022 Rapyuta Robotics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import typing
+
+from munch import munchify, unmunchify
+from rapyuta_io import Client
+
+from riocli.jsonschema.validate import load_schema
+from riocli.model import Model
+
+
+class UserGroup(Model):
+
+    def find_object(self, client: Client) -> typing.Any:
+        group_guid, usergroup = self.rc.find_depends({'kind': self.kind.lower(), 'nameOrGUID': self.metadata.name,
+                                                      'organization': self.metadata.organization})
```

Comment:
Format this section. 
```suggestion
        group_guid, usergroup = self.rc.find_depends({
            'kind': self.kind.lower(),
            'nameOrGUID': self.metadata.name,
            'organization': self.metadata.organization
        })
```